caravel/assets/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "caravel",
-  "version": "0.1.0",
+  "version": "0.10.1",
```

Comment:
should package.json version be the same as caravel version? using this method we would need to remember update the package.json version with any js change. this would invalidate the cache on ALL of the js files and not just on the files that did change. not quite what we want to do here.

essentially, we want the filename to update if the file changed, and to stay the same if the contents did not change.

here's a good article on how to do this using webpack: https://medium.com/@okonetchnikov/long-term-caching-of-static-assets-with-webpack-1ecb139adb95